src/libs/StepperMotor.h

```diff
@@ -69,6 +71,7 @@ class StepperMotor  : public Module {
         float steps_per_mm;
         float max_rate; // this is not really rate it is in mm/sec, misnamed used in Robot and Extruder
         float acceleration;
+        float pressure_advance;
```

Comment:
I agree.  I didn't want to put it there, but couldn't figure out how to access the extruder pressure advance value from within the Block context.  Can you suggest a way to get around the scoping issues?